src/share/classes/com/sun/javatest/regtest/config/ExtraPropDefns.java

```diff
@@ -221,6 +223,9 @@ private List<String> compile(JDK jdk, Path classDir, SearchPath classpath,
         }
 
         if (needCompilation) {
+            if (trace) {
+                System.err.println("Compiling extra property definition files: " + javacArgs);
```

Comment:
Could you also log why the compilation was needed? i.e. when setting needCompilation, log sf and cf.

Reply:
Will investigate, as a separate RFE

Reply:
https://bugs.openjdk.org/browse/CODETOOLS-7903329